offchain-modules/packages/scripts/src/utils/eth_batch_test.ts

```diff
@@ -162,9 +166,13 @@ export async function lock(
   }
 
   for (let i = 0; i < batchNum; i++) {
-    const lockTxHash = (await provider.sendTransaction(signedLockTxs[i])).hash;
-    await asyncSleep(intervalMs);
-    lockTxHashes.push(lockTxHash);
+    try {
+      const lockTxHash = (await provider.sendTransaction(signedLockTxs[i])).hash;
+      await asyncSleep(intervalMs);
+      lockTxHashes.push(lockTxHash);
+    } catch (e) {
+      logger.error(e.stack);
+    }
```

Comment:
should throw e in catch block ?

Reply:
added throw e, and stop running, troubleshoot errors manually